include/sys/atomic.h

```diff
@@ -68,6 +68,14 @@ typedef void *atomic_ptr_t;
  * INTERNAL_HIDDEN @endcond
  */
 
+/**
+ * @brief This macro computes the number of atomic variables necessary to
+ * represent at bitmap with @a num_bits.
+ *
+ * @param num_bits Number of bits.
+ */
+#define ATOMIC_BITMAP_SIZE(num_bits) (1 + ((num_bits) - 1) / ATOMIC_BITS)
```

Comment:
Merge this with the commit that modifies the doxyfile

Reply:
done